### PR TITLE
Update utils2.ts

### PR DESCRIPTION
**Pull request by Dylan0916**

### CODE REVIEW

**Comment by dylan-funnow** [comment]
test1

**Comment by dylan-funnow** [comment]
test1

File: src/utils2.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export function log1() {
-  console.log('=== log1 ===');
+  console.log('=== log2 ===');
```
Comment (dylan-funnow) [inline src/utils2.ts:2]: test1

**Comment by dylan-funnow** [comment]
test1

File: src/utils2.ts (hunk 1 above)
Comment (dylan-funnow) [inline src/utils2.ts:2]: test1

Comment (Dylan0916) [reply src/utils2.ts:2]: test2

Comment (dehao0916) [reply src/utils2.ts:2]: B1

Comment (Dylan0916) [reply src/utils2.ts:2]: A2

Comment (Dylan0916) [reply src/utils2.ts:2]: A1

Comment (Dylan0916) [reply src/utils2.ts:2]: A1

Comment (Dylan0916) [reply src/utils2.ts:2]: A1

Comment (Dylan0916) [reply src/utils2.ts:2]: A2

Comment (dylan-funnow) [inline src/utils2.ts:2]: C1

Comment (dylan-funnow) [inline src/utils2.ts:2]: 1

Comment (dehao0916) [reply src/utils2.ts:2]: 2

Comment (Dylan0916) [reply src/utils2.ts:2]: 3

Comment (Dylan0916) [reply src/utils2.ts:2]: 4

**Comment by Dylan0916** [comment]
dylan test

**Comment by dylan-funnow** [comment]
> dylan test

C replied

**Comment by Dylan0916** [comment]
> C replied

**Pull request closed without merging**